altair/vegalite/v2/examples/connected_scatterplot.py

```diff
@@ -18,8 +18,7 @@
 
 points = alt.Chart(driving).mark_circle().encode(
     alt.X('miles', scale=alt.Scale(zero=False)),
-    alt.Y('gas', scale=alt.Scale(zero=False)),
-    order='year'
```

Comment:
The order is really important here, because it makes the line connect points in temporal order. We should leave this in here.

Reply:
Ah, strike that, I see what you did. Order is not necessary for points :smile:

Reply:
Glad to hear I'm not crazy!